benchmarks/mysql-kafka-cdc/benchi.yml

```diff
@@ -0,0 +1,104 @@
+infrastructure:
+  kafka:
+    compose: ../../shared/kafka/compose.yml
+  mongo:
+    compose: ../../shared/mysql/compose.yml
+
+tools:
+  kafka-connect:
+    compose:
+      - ../../shared/kafka-connect/compose.yml
+      - ./kafka-connect/compose.override.yml
+  conduit:
+    compose:
+      - ../../shared/conduit/compose.yml
+      - ./conduit/compose.override.yml
+
+metrics:
+  conduit_metrics:
+    collector: conduit
+    tools:
+      - conduit
+    settings:
+      url: http://localhost:8080/metrics
+  kafka_metrics_for_conduit:
+    collector: kafka
+    tools:
+      - conduit
+    settings:
+      url: http://localhost:7071/metrics
+      topics: [ "conduit.users" ]
```

Comment:
If you name this topic `conduit.conduit.users` (or, alternatively, `conduit.testdb.users`), then you can have one metric for both tools. That will make them appear as one column in the results.

Reply:
went with `mysql.testdb.users`